Refresh.GameServer/Endpoints/Game/ResourceEndpoints.cs

```diff
@@ -32,6 +33,9 @@ public class ResourceEndpoints : EndpointGroup
         if (gameAsset == null)
             return BadRequest;
 
+        long earliestTime = new DateTimeOffset(2007, 1, 1, 0, 0, 0, TimeSpan.Zero).ToUnixTimeSeconds();
```

Comment:
This should probably be a constant/static variable somewhere

Maybe `EarliestDate` on `IDateTimeProvider`